src/io/filesystem.ts

```diff
@@ -194,6 +214,11 @@ export async function sampleProject(projectPath: string) {
   }
 }
 
+let currentProjectId = '';
+export function setCurrentProjectId(projectId: string) {
```

Comment:
Sharp eyes @danvk :-). 

I wasn't sure (still not 100% sure) how to make the current project id available throughout the app. 
I started with storing the id in the project store, but then figured out that for the filesystem calls would be **much** simpler to just save the `currentProjectId` and use it for the "web".

With the "unification" of desktop and web I think that this will be cleaner in your next review.